src/data/machines.js

```diff
@@ -139,32 +138,180 @@ export const ephemeralExternalIpAddressPrice = { // per hour in dollars for all
   preemptible: 0.002
 }
 
-// Prices below are per GB per https://cloud.google.com/compute/all-pricing#disk
-export const regionToDiskPrice = [
-  { name: 'ASIA-EAST1', monthlyDiskPrice: 0.04 },
-  { name: 'ASIA-EAST2', monthlyDiskPrice: 0.05 },
-  { name: 'ASIA-NORTHEAST1', monthlyDiskPrice: 0.052 },
-  { name: 'ASIA-NORTHEAST2', monthlyDiskPrice: 0.052 },
-  { name: 'ASIA-NORTHEAST3', monthlyDiskPrice: 0.052 },
-  { name: 'ASIA-SOUTH1', monthlyDiskPrice: 0.048 },
-  { name: 'ASIA-SOUTHEAST1', monthlyDiskPrice: 0.044 },
-  { name: 'ASIA-SOUTHEAST2', monthlyDiskPrice: 0.052 },
-  { name: 'AUSTRALIA-SOUTHEAST1', monthlyDiskPrice: 0.054 },
-  { name: 'EUROPE-NORTH1', monthlyDiskPrice: 0.044 },
-  { name: 'EUROPE-WEST1', monthlyDiskPrice: 0.04 },
-  { name: 'EUROPE-WEST2', monthlyDiskPrice: 0.048 },
-  { name: 'EUROPE-WEST3', monthlyDiskPrice: 0.048 },
-  { name: 'EUROPE-WEST4', monthlyDiskPrice: 0.044 },
-  { name: 'EUROPE-WEST6', monthlyDiskPrice: 0.052 },
-  { name: 'NORTHAMERICA-NORTHEAST1', monthlyDiskPrice: 0.044 },
-  { name: 'SOUTHAMERICA-EAST1', monthlyDiskPrice: 0.06 },
-  { name: 'US-CENTRAL1', monthlyDiskPrice: 0.04 },
-  { name: 'US-EAST1', monthlyDiskPrice: 0.04 },
-  { name: 'US-EAST4', monthlyDiskPrice: 0.044 },
-  { name: 'US-WEST1', monthlyDiskPrice: 0.04 },
-  { name: 'US-WEST2', monthlyDiskPrice: 0.048 },
-  { name: 'US-WEST3', monthlyDiskPrice: 0.048 },
-  { name: 'US-WEST4', monthlyDiskPrice: 0.044 }
+// Disk prices below are per GB per https://cloud.google.com/compute/all-pricing#disk
+// GPU prices below are hourly and per GPU (https://cloud.google.com/compute/gpus-pricing).
+// The data below comes from Google's pricing API: https://cloud.google.com/billing/v1/how-tos/caltalog-api
+// There is a script available in https://gist.github.com/wnojopra/be5be3ab7e6a09a351b89085992a940d
```

Comment:
Let's put a TODO above to update the URL after merging the `dsp-scripts` PR.

Reply:
Done.